package.json

```diff
@@ -32,7 +32,7 @@
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks",
   "devDependencies": {
     "adm-zip": "0.4.13",
-    "azure-devops-node-api": "^10.2.2",
+    "azure-devops-node-api": "^12.2.0",
```

Comment:
Required for test plans API : https://github.com/microsoft/azure-devops-node-api/blob/master/api/TestPlanApi.ts in version 12.2.0